### PR TITLE
CIVDT-196: Added Trader Address functionality

### PR DESCRIPTION
**Pull request by dannybassi**

### CODE REVIEW

File: test/connectors/ImporterAddressConnectorSpec.scala [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  val errorModel: HttpResponse = HttpResponse(Status.NOT_FOUND, "Error Message")
+
+  val traderAddress: TraderAddress = TraderAddress("first", "second", Some("third"), "fourth")
+
+  object Connector extends ImporterAddressConnector(mockHttp, appConfig)
+
+  "Importer Address Connector" should {
+
+    def getAddressResult(): Future[HttpGetResult[TraderAddress]] = Connector.getAddress("1")
```
Comment (tvaidotas) [inline test/connectors/ImporterAddressConnectorSpec.scala:41]: "1" is used multiple times, should become a variable

File: test/connectors/ImporterAddressConnectorSpec.scala [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import uk.gov.hmrc.http.HttpResponse
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class ImporterAddressConnectorSpec extends SpecBase with MockHttp {
+
+  val errorModel: HttpResponse = HttpResponse(Status.NOT_FOUND, "Error Message")
+
+  val traderAddress: TraderAddress = TraderAddress("first", "second", Some("third"), "fourth")
```
Comment (tvaidotas) [inline test/connectors/ImporterAddressConnectorSpec.scala:35]: TraderAddress("first", "second", Some("third"), "fourth") is used in multiple files, should be a global variable

File: test/connectors/ImporterAddressConnectorSpec.scala [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import play.api.http.Status
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import uk.gov.hmrc.http.HttpResponse
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class ImporterAddressConnectorSpec extends SpecBase with MockHttp {
+
+  val errorModel: HttpResponse = HttpResponse(Status.NOT_FOUND, "Error Message")
```
Comment (tvaidotas) [inline test/connectors/ImporterAddressConnectorSpec.scala:33]: val errorModel: HttpResponse = HttpResponse(Status.NOT_FOUND, "Error Message") is used in multiple files, should be a global variable

File: test/connectors/httpParsers/ImporterAddressHttpParserSpec.scala [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import play.api.http.Status
+import play.api.libs.json.{JsObject, Json}
+import uk.gov.hmrc.http.HttpResponse
+
+class ImporterAddressHttpParserSpec extends SpecBase {
+
+  val errorModel: HttpResponse = HttpResponse(Status.NOT_FOUND, "Error Message")
+
+  val traderAddressJson: JsObject = Json.obj(
```
Comment (tvaidotas) [inline test/connectors/httpParsers/ImporterAddressHttpParserSpec.scala:31]: is used in multiple files, should be a global variable

**Review by tvaidotas — CHANGES_REQUESTED** [review]

File: test/connectors/ImporterAddressConnectorSpec.scala (hunk 1 above)
Comment (dannybassi) [reply test/connectors/ImporterAddressConnectorSpec.scala:41]: complete

File: test/connectors/ImporterAddressConnectorSpec.scala (hunk 2 above)
Comment (dannybassi) [reply test/connectors/ImporterAddressConnectorSpec.scala:35]: complete

File: test/connectors/httpParsers/ImporterAddressHttpParserSpec.scala (hunk 4 above)
Comment (dannybassi) [reply test/connectors/httpParsers/ImporterAddressHttpParserSpec.scala:31]: complete

File: test/connectors/ImporterAddressConnectorSpec.scala (hunk 3 above)
Comment (dannybassi) [reply test/connectors/ImporterAddressConnectorSpec.scala:33]: complete

**Review by tvaidotas — APPROVED** [review]

**Pull request merged**